pandas/tests/io/test_feather.py

```diff
@@ -247,3 +248,20 @@ def test_string_inference(self, tmp_path):
             result = read_feather(path)
         expected = pd.DataFrame(data={"a": ["x", "y"]}, dtype="string[pyarrow_numpy]")
         tm.assert_frame_equal(result, expected)
+
+    def test_out_of_bounds_datetime_to_feather(self):
+        # GH#47832
+        df = pd.DataFrame(
+            {
+                "date": [
+                    datetime.fromisoformat("1654-01-01"),
+                    datetime.fromisoformat("1920-01-01"),
+                ],
+            }
+        )
+
+        feather_path = "tmp.feather"
```

Comment:
This will leave behind a file after running the test which is undesirable. See the test above and use that pattern to avoid this.